instance/templates/instance/haproxy/openedx.conf

```diff
@@ -1,10 +1,10 @@
 {% autoescape off %}
     # Backend configuration for {{ domain }}
-    stick-table type string len 128 size 20k expire 12h
-    stick on req.cook(sessionid)
+    cookie openedx-backend insert postonly indirect
```

Comment:
This only sets the cookie on POST requests, since I expected it only to be relevant for logged in users.  You had some issues with the static assets, so maybe we need this for all users.  We can change `postonly` to `nocache` to set the cookies for anonymous users as well.

Reply:
I think we should make it sticky for all users. Do you see any potential downsides to it?

If such a mismatched static asset problem happens again, we definitely don't want to discover that kind of failure the same way we did this time, and I would rather we just take the extra hit of clients possibly downloading the static assets multiple times without us noticing for a while in the worst case.